DESCRIPTION

```diff
@@ -44,8 +45,7 @@ Config/testthat/edition: 3
 License: MIT + file LICENSE
 Remotes:
     epiforecasts/socialmixr
-Imports: 
-    checkmate,
+Imports:
```

Comment:
Not a comment on the change, but is there a reason socialmixr is being installed from github, the version on CRAN matches that on github?

Reply:
Thanks for noticing this, we were using a dev version of `socialmixr` for a while for some reason I cannot remember. Now removed in https://github.com/epiverse-trace/finalsize/pull/73/commits/1695234c584728fdb46fd26eb9d9569ca1003163